tests/test_utils.py

```diff
@@ -313,3 +313,19 @@ def test_check_status_code_for_missing_status_code(capsys, repo_url):
 
     captured = capsys.readouterr()
     assert captured.out == "NOTE: Try --verbose to see response content.\n"
+
+
+@pytest.mark.parametrize(
+    ("filename, expected"),
+    [
+        (helpers.WHEEL_FIXTURE, "15.4 KB"),
+        (helpers.SDIST_FIXTURE, "20.8 KB"),
+        (helpers.NEW_SDIST_FIXTURE, "26.1 KB"),
+        (helpers.NEW_WHEEL_FIXTURE, "21.9 KB"),
+    ],
+)
+def test_get_file_size(filename, expected):
+    """Compare the size of a file to the expected value."""
+    file_size = utils.get_file_size(filename)
+
+    assert file_size == expected
```

Comment:
Nice job with parameterization. However, this doesn't cover the "MB" case ([see the report](https://codecov.io/gh/pypa/twine/compare/89aab15175a89d120802756ed4399d7dd93959bc...3954ad972e062e20f3b132789386ba6d3994c5f8/src/twine/utils.py?before=twine/utils.py)). That said, I'd rather not add a 1 MB file to the repo just to test this.

So, instead using actual files, how about changing the parameters to be a size in bytes and the expected string, then using pytest's [`monkeypatch` fixture](https://docs.pytest.org/en/latest/monkeypatch.html) to make `os.path.getsize` return the parameterized size in bytes?